client/src/main/java/org/spine3/protobuf/TypeToClassMap.java

```diff
@@ -74,7 +74,12 @@
      *
      * <p>For example, for a key {@code spine.base.EventId}, there will be the value {@code org.spine3.base.EventId}.
      */
-    private static final BiMap<TypeName, ClassName> NAMES_MAP = Builder.build();
+    private static final BiMap<TypeName, ClassName> TYPE_TO_CLASS_MAP = Builder.build();
+
+    /**
+     * The inverse view of the primary type to class map.
+     */
+    private static final BiMap<ClassName, TypeName> CLASS_TO_TYPE_MAP = TYPE_TO_CLASS_MAP.inverse();
```

Comment:
Assuming the above, I'd say that `underlyingBiMap` or just `biMap` would be a good candidate for what is now called `TYPE_TO_CLASS_MAP`.